### ISSUE TITLE
Tensorflow Estimator Examples

### ISSUE THREAD

**Issue by sweaterr** [sweaterr/readinglist#303]
https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/learn

**Comment by sweaterr** [comment]
https://medium.com/onfido-tech/higher-level-apis-in-tensorflow-67bfb602e6c0

**Comment by sweaterr** [comment]
Estimator는 다음이 필요하다
* parameters
* configuration
* model function
  * first-class function => 모델 재생성에 좋다.
  * takes
    * input features
    * labels
    * mode
    * hyperparameters
  * return
    * EstimatorSpec: 예측,손실,훈련,평가 op들을 입력받아서 그래프를 정의함

**Comment by sweaterr** [comment]
Experiment takes
* estimator
* Train and evaluation data
  * first-class function
* Training and Evaluating hooks.
